linux/guest_customization/linux_gosc_workflow.yml

```diff
@@ -15,13 +15,19 @@
     - include_tasks: ../setup/test_setup.yml
       vars:
         skip_test_no_vmtools: True
-        deploy_router_vm: "{{ router_vm_required | default(False) }}"
 
+    # Check cloud-init version for cloud-init gosc workflow
+    - include_tasks: ../utils/cloudinit_pkg_check.yml
     - include_tasks: check_gosc_support_status.yml
+    # Prepare router VM, vSwitch and portgroup
+    - include_tasks: ../../common/network_testbed_setup.yml
```

Comment:
I think `network_testbed_setup.yml` can be moved to before
`- name: Initialize the GOS customization spec`, and after photon os upgrading cloud-init and rhel family installed perl package, in case package installation failure ends the test case. And `cloudinit_pkg_check.yml` shall be executed after Photon OS installing/upgrading cloud-init.